examples/models/deepseek_v3_2/deepseek_v3_2_decode_front_scope123_int8_quant.py

```
@@ -0,0 +1,1108 @@
+        k_idx[b : b + 1, half:QK_ROPE_HEAD_DIM] = k_hi * cos_hi + k_lo * sin_hi
+
+    q_idx_full = q_view.reshape(BATCH, INDEX_HEADS * INDEX_HEAD_DIM)
+    weights = (hidden_states @ weights_proj.to(torch.bfloat16).float()) * (
+        INDEX_HEADS ** -0.5 * INDEX_HEAD_DIM ** -0.5
+    )
+    q_idx_full_pre_quant = q_idx_full.clone()
+    k_idx_pre_quant = k_idx.clone()
+    q_idx_full = int8_quant_dequant_groups(q_idx_full)
+    k_idx = int8_quant_dequant_groups(k_idx)
+    q_idx_i8_golden, q_idx_scale_golden = int8_quant_groups(q_idx_full_pre_quant)
+    k_idx_i8_golden, k_idx_scale_golden = int8_quant_groups(k_idx_pre_quant)
```

Ruff (0.15.11)

[warning] Unpacked variable `k_idx_scale_golden` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)